auth.go

```diff
@@ -51,6 +51,7 @@ func NewError(base error, detail string) error {
 // A Permit is an opaque binary string that encodes domain-specific roles.
 // This could be a string of bits with special meanings, or a blob of JSON, etc.
 type Permit struct {
+	name string
```

Comment:
Looking at this, it is completely unused, I do not know why i never deleted it. It shouldnt be part of anything. Mistake on my part.